### PR TITLE
update implementation

### PR DESCRIPTION
**Pull request by loomis**
Update the implementation of the proxy by refining the error handling and refactoring to allow a direct connection from the CIMI server to dataClay eventually (not currently working).

### CODE REVIEW

**Review by cjdcordeiro — CHANGES_REQUESTED** [review]
@loomis I can't build it:

```
Step 12/16 : ADD ./target/proxy-0.1.0-SNAPSHOT-standalone.jar /root/dataclay/proxy.jar
ADD failed: stat /var/lib/docker/tmp/docker-builder960612566/target/proxy-0.1.0-SNAPSHOT-standalone.jar: no such file or directory
```

File: dataclay-proxy/project.clj [hunk 1]
```diff
@@ -14,10 +14,10 @@
 ;; permissions and limitations under the License.
 ;;
 
-(def +version+ "1.10-SNAPSHOT")
-(def +slipstream-version+ "3.53")
+(def +version+ "2.4")
+(def +slipstream-version+ "3.67")
 
-(defproject com.sixsq.dataclay/proxy "0.1.0-SNAPSHOT"
```
Comment (cjdcordeiro) [inline dataclay-proxy/project.clj:20]: I think this is the faulty line

**Comment by loomis** [comment]
@cjdcordeiro I've fixed the bad reference to the jar file.  It should build correctly now.  I've also removed the old wrapper and support scripts.

**Review by cjdcordeiro — APPROVED** [review]
tested and working

**Pull request merged**